x-pack/plugins/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -610,18 +606,20 @@ export class ObservabilityAIAssistantClient {
     messages,
     connectorId,
     signal,
+    responseLanguage,
   }: {
     messages: Message[];
     connectorId: string;
     signal: AbortSignal;
+    responseLanguage: string;
   }) => {
     const response$ = await this.chat('generate_title', {
       messages: [
         {
           '@timestamp': new Date().toString(),
           message: {
             role: MessageRole.System,
-            content: `You are a helpful assistant for Elastic Observability. Assume the following message is the start of a conversation between you and a user; give this conversation a title based on the content below. DO NOT UNDER ANY CIRCUMSTANCES wrap this title in single or double quotes. This title is shown in a list of conversations to the user, so title it for the user, not for you.`,
+            content: `You are a helpful assistant for Elastic Observability. Assume the following messages are a conversation between you and a user; give this conversation a title based on the content below. DO NOT UNDER ANY CIRCUMSTANCES wrap this title in single or double quotes. This title is shown in a list of conversations to the user, so title it for the user, not for you. Please create the title in ${responseLanguage}.`,
```

Comment:
The conversation messages are concatenated into one message, right? Original seems correct?

Reply:
I reverted this change.